installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -39,7 +39,7 @@ include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
 
 .Additional resources
 
-* See link:https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.hcpa6/bhslzvs.htm[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in the IBM Knowledge Center.
+* See link:https://www.ibm.com/docs/en/zvm/7.1?topic=networks-bridging-hipersockets-lan-zvm-virtual-switch[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in IBM Documentation.
```

Comment:
I don't think that "Documentation" needs to be capitalized.

Reply:
It should be capitalized "IBM Knowledge Center" was renamed to "IBM Documentation" 

See here: https://www.ibm.com/docs/en